src/main/java/org/codehaus/plexus/archiver/tar/PlexusIoTBZ2FileResourceCollection.java

```diff
@@ -0,0 +1,14 @@
+package org.codehaus.plexus.archiver.tar;
+
+import javax.inject.Named;
+
+import java.io.File;
+
+/**
+ * Alias for {@link PlexusIoTarBZip2FileResourceCollection}
+ */
+@Named( "tbz2" )
+public class PlexusIoTBZ2FileResourceCollection
+    extends PlexusIoTarBZip2FileResourceCollection
+{
+}
```

Comment:
`tgz` contains a new `newTarFile` method, this one does not. Why?

Reply:
Found the leftover, removed